pkg/kv/kvnemesis/watcher.go

```diff
@@ -192,6 +193,44 @@ func (w *Watcher) processEvents(ctx context.Context, eventC chan kvcoord.RangeFe
 					}
 				}
 				w.mu.Unlock()
+			case *roachpb.RangeFeedSSTable:
+				w.mu.Lock()
+				if len(e.Data) == 0 {
+					return errors.AssertionFailedf("no SST data found")
+				}
+				// TODO(erikgrinaker): This should handle range keys too.
+				iter, err := storage.NewMemSSTIterator(e.Data, false /* verify */, storage.IterOptions{
+					KeyTypes:   storage.IterKeyTypePointsOnly,
+					LowerBound: keys.MinKey,
+					UpperBound: keys.MaxKey,
+				})
+				if err != nil {
+					return err
+				}
+				defer iter.Close()
+				for iter.SeekGE(storage.MVCCKey{Key: keys.MinKey}); ; iter.Next() {
+					if ok, err := iter.Valid(); err != nil {
+						return err
+					} else if !ok {
+						break
+					}
+					key := iter.Key()
+					rawValue := iter.Value()
+					mvccValue, err := storage.DecodeMVCCValue(rawValue)
+					if err != nil {
+						return err
+					}
+					mvccValue.Value.Timestamp = key.Timestamp
+					if seq := mvccValue.KVNemesisSeq.Get(); seq > 0 {
+						w.env.Tracker.Add(key.Key, nil, key.Timestamp, seq)
+					}
+					if err := w.handleValueLocked(ctx, roachpb.Span{Key: key.Key}, mvccValue.Value, nil); err != nil {
```

Comment:
Could you add a comment on `(*SeqTracker).Add` explaining that for point writes, we rely on a `RangefeedValueHeaderFilter` knob to tell us the seqnos of writes as they get emitted from the rangefeed, while for AddSST the emitted rangefeed event itself contains the seqno and so it is added in the watcher?
This is a leaky comment but I think it tells people everything they need to know.